### DISCUSSION TITLE
Tell user if `minimumReleaseAge` is non-functional (docs and PR body text)

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#30666] [Suggest an Idea]
### Tell us more.

A user said: [^1]

> at minimum, it should be  clear when `minimumReleaseAge` is ignored because the dep manager lacks support for the feature (perhaps on the Dependency Dashboard).

How about:

- Listing the datasources _with_ timestamp in the `minimumReleaseAge` docs?
- Printing a message in Renovate's PR body when `minimumReleaseAge` is non-functional?

[^1]: [Comment from `nipunn1313`](https://github.com/renovatebot/renovate/issues/17764#issuecomment-1704568613)

**Comment by RahulGautamSingh** [comment]
We have `releaseTimestamp` field in the docs for each datasource.

**Reply by HonkingGoose** [reply]
You're right! But that means you have to click away from the `minimumReleaseAge` docs to read the datasource docs. And worse, you must click on each datasource that you're using, to know if it has that `releaseTimeStamp` field.

How about updating our docs generating script, so it:

1. Collects all datasources with `releaseTimeStamp` field.
1. Makes a list.
1. Puts the list in the `minimumReleaseAge` docs, with a good heading like `#### Datasources with timestamp`.

**Reply by RahulGautamSingh** [reply]
Sounds better, than the previous suggestion of including this info in the pr.

I think we can create an issue now.

**Reply by HonkingGoose** [reply]
See issue:

- #30714
